static/report/js/loopalyzer.js

```diff
@@ -0,0 +1,52 @@
+// Moves backward one day
+function loopalyzerBackward() {
+    var moment = window.moment;
+    var from = moment($("#rp_from").val()).subtract(1,'day');
+    var to = moment($("#rp_to").val()).subtract(1,'day');
+    $("#rp_from").val(from.format('YYYY-MM-DD'));
+    $("#rp_to").val(to.format('YYYY-MM-DD'));
+    $("#rp_show").click();
+}
+
+// Moves backward same amount as shown (e.g. whole week)
+function loopalyzerMoreBackward() {
+    var moment = window.moment;
+    var from = moment($("#rp_from").val())
+    var to = moment($("#rp_to").val())
+    var diff = to.diff(from, 'days') + 1;
+    from.subtract(diff, 'days');
+    to.subtract(diff, 'days');
+    $("#rp_from").val(from.format('YYYY-MM-DD'));
+    $("#rp_to").val(to.format('YYYY-MM-DD'));
+    $("#rp_show").click();
+}
+
+// Moves forward one day
+function loopalyzerForward() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['loopalyzerForward' is defined but never used. (no-unused-vars)](https://app.codacy.com/app/m-kozak/cgm-remote-monitor/pullRequest?prid=1196151)